stable_baselines/common/segment_tree.py

```diff
@@ -98,33 +112,46 @@ def sum(self, start=0, end=None):
     def find_prefixsum_idx(self, prefixsum):
         """
         Find the highest index `i` in the array such that
-            sum(arr[0] + arr[1] + ... + arr[i - i]) <= prefixsum
+            sum(arr[0] + arr[1] + ... + arr[i - i]) <= prefixsum for each entry in prefixsum
 
         if array values are probabilities, this function
         allows to sample indexes according to the discrete
         probability efficiently.
 
-        :param prefixsum: (float) upperbound on the sum of array prefix
-        :return: (int) highest index satisfying the prefixsum constraint
+        :param prefixsum: (numpy float) upper bounds on the sum of array prefix
```

Comment:
These should be numpy ndarrays, correct? If so, update docs to reflect that.

Reply:
I changed it to (ndarray) but was not sure what exactly to write. Is there some guideline on how to specify objects from external libraries like numpy?

Reply:
`np.ndarray` would be the right way, and because we are using python >= 3.5 you can directly use type hints in the signature (we use `py.type` to check)